src/VisualStudio/CSharp/Impl/LanguageService/CSharpCodeCleanupFixer.cs

```diff
@@ -383,12 +386,23 @@ async Task<Solution> ApplyFixAsync(ProgressTracker progressTracker, Cancellation
             }
         }
 
-        private async Task<bool> FixAsync(Workspace workspace, Func<ProgressTracker, CancellationToken, Task<Solution>> applyFixAsync, ICodeCleanUpExecutionContext context, CancellationToken cancellationToken)
+        private async Task<bool> FixAsync(Workspace workspace, Func<ProgressTracker, CancellationToken, Task<Solution>> applyFixAsync, ICodeCleanUpExecutionContext context,
+            string contextName, CancellationToken cancellationToken)
```

Comment:
@olegtk 
I think we already talked about it once. are you guys going to remove CancellationToken from ICodeCleanUpFixer API
http://source.roslyn.io/#Microsoft.VisualStudio.LanguageServices/Implementation/CodeCleanup/CodeCleanUpFixer.cs,16

right now, things are a bit weird since we get both OperationContext and CancellationToken. and we need to create this linked cancellation token.